src/client/renderer/shaders/particle_corona_vs.glsl

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
I would be in favor of adding some folder structure to our shaders. In fact, I have a (now horribly stale and in need of closing) pull request that moves our includes to "include". So, if we're going to have an assortment of particle shaders, we could put those in "shaders/particle" or something.